locksmith/src/operations/userMetadataOperations.ts

```diff
@@ -52,6 +52,26 @@ export async function getMetadata(
   return data ? data.data : data
 }
 
+export async function getUserEmailRecipient({
+  lockAddress,
+  ownerAddress,
+}: {
+  lockAddress: string
+  ownerAddress: string
+}): Promise<string | undefined> {
+  const ownerMetadata = await getMetadata(
```

Comment:
Does this only get metadata? 🤔 You should only use function which makes a database query.

Reply:
yes, this gets only user metadata with a query